test/index.test.ts

```diff
@@ -312,3 +312,35 @@ test('API fetch poll Tweet', async () => {
   expect(choices[3].count).toEqual(31706);
   expect(choices[3].percentage).toEqual(58);
 });
+
+test('API fetch user', async () => {
+  const result = await cacheWrapper(
+    new Request('https://api.fxtwitter.com/wazbat', {
+      method: 'GET',
+      headers: botHeaders
+    })
+  );
+  expect(result.status).toEqual(200);
+  const response = (await result.json()) as UserAPIResponse;
+  expect(response).toBeTruthy();
+  expect(response.code).toEqual(200);
+  expect(response.message).toEqual('OK');
+
+  const user = response.user as APIUser;
+  expect(user).toBeTruthy();
+  expect(user.url).toEqual('https://twitter.com/wazbat');
+  expect(user.id).toEqual('157658332');
+  expect(user.name).toEqual('Wazbat');
```

Comment:
Maybe we should leave out name and location as those are subject to change meanwhile screen_name and the others should be relatively static

Reply:
Otherwise our unit tests will literally be depending on you never updating those aspects of your profile lol